storage/redis/peer_store.go

```diff
@@ -785,16 +779,16 @@ func (ps *peerStore) collectGarbage(cutoff time.Time) error {
 				//_, err := conn.Do("DEL", ihStr)
 
 				_ = conn.Send("MULTI")
-				_ = conn.Send("HDEL", group, ihStr)
+				_ = conn.Send("HDEL", "I"+af, ihStr)
 				if isSeeder {
-					_ = conn.Send("DECR", ps.infohashCountKey(group))
+					_ = conn.Send("DECR", "I"+af)
```

Comment:
Something about these few lines seems wrong. We call `HDEL` and `DECR` on the same key, which I find pretty dubious.
Tbh I don't have the structure of all this stuff in redis in mind, so I don't know which one is correct, but one of them is probably not correct :)

Reply:
This was definitely a bug. I found another one as well, so I created an addressFamily type that helps build these keys. All the methods are small enough that they should be inlined, so zero-cost.